lte/gateway/docker/python-precommit/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM python:3.8-alpine as builder
-
-RUN addgroup -S linter && adduser -S -G linter linter
-RUN apk add --no-cache git gcc musl-dev # temp
-
-COPY lte/gateway/docker/python-precommit/requirements.txt /
-RUN pip wheel -r /requirements.txt --wheel-dir=/wheelhouse/
+FROM python:3.9.10-alpine
 
-
-FROM python:3.8-alpine as runner
+ENV WPS_VERSION='0.17.0'
 
 RUN addgroup -S linter && adduser -S -G linter linter
 
-RUN apk add --no-cache jq
-
-COPY --from=builder /wheelhouse/ /wheelhouse/
-RUN pip install /wheelhouse/*
+# Installing wemake-python-styleguide==0.17.0 is not
+# working with a requirements.in file.
+# hadolint ignore=DL3013
+RUN pip install --no-cache-dir "wemake-python-styleguide==$WPS_VERSION" \
+                "add-trailing-comma" \
+                "isort" \
+                "autopep8"
```

Comment:
I don't think we use quotation marks for dependencies in `RUN pip install` commands anywhere else in the project. I suggest removing the marks. 

I don't think we need to create an `WPS_VERSION` env variable either if we are only going to use it once. That would probably allow you to remove the lint ignore line.

Reply:
Removed the quotation marks and in-lined the version. The ignore is still needed - see reviewdog comment below.